codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -296,6 +296,7 @@ final DefaultPropertyKey verifyKey(PropertyKey key) {
         private int totalChildWeights;
         private int prioritizableForTree = 1;
         private boolean resetSent;
+        private boolean headerSent;
```

Comment:
Right, I'm not against it ... I just don't want to make that sort of interface change now. That's something we want to spend more time thinking about to get it right.